CONTRIBUTING.rst

```diff
@@ -0,0 +1,1258 @@
+##################################
+|TITLE| (|DATE|)
+##################################
+
+.. |TITLE| replace:: OCA Guidelines
+.. |DATE| replace:: 2018/08/24
+.. |RELATED| replace:: OCA Guidelines
```

Comment:
If we do not use the footer section then either we remove the variable or we add this part in the header section (RELATED/CATEGORIES).